BrainPortal/lib/view_scopes.rb

```diff
@@ -189,7 +189,7 @@ def initialize_copy(other)
     #
     # Note that pagination is not applied by default; specify +paginate+ to
     # paginate +collection+.
-    def apply(collection, on_failure: :ignore, paginate: false)
+    def apply(collection, on_failure: :ignore, paginate: false, sorting: true)
```

Comment:
Done; I will wait until we've finished the other discussion about the to_a() in the other file before I commit.